### PR TITLE
Upgrade stylelint to v15

### PR DESCRIPTION
**Pull request by bryanjtc**
Closes: #1373 
Updated stylelint tests
Upgraded and matched pnpm version in package.json and gh action

### CODE REVIEW

**Comment by bryanjtc** [comment]
@sobolevn Can you check this?

File: packages/stylelint-config-scss/package.json [hunk 1]
```diff
@@ -20,20 +20,20 @@
...
+    "stylelint-config-standard": "33.0.0",
+    "stylelint-csstree-validator": "github:bryanjtc/stylelint-validator#upgrade-package",
     "stylelint-declaration-block-no-ignored-properties": "2.7.0",
     "stylelint-declaration-strict-value": "1.9.2",
     "stylelint-high-performance-animation": "1.8.0",
     "stylelint-no-indistinguishable-colors": "github:bryanjtc/stylelint-no-indistinguishable-colors#upgrade-package",
     "stylelint-order": "6.0.3",
     "stylelint-scss": "5.0.0",
-    "stylelint-selector-tag-no-without-class": "2.0.5",
+    "stylelint-selector-tag-no-without-class": "github:bryanjtc/stylelint-selector-tag-no-without-class#upgrade-package",
```
Comment (sobolevn) [inline packages/stylelint-config-scss/package.json:36]: You had to fork it? If so, we can just remove this rule - it is better in the long run.

**Review by sobolevn — APPROVED** [review]
Thank you! 🎉

**Comment by sobolevn** [comment]
Release is in the works

**Comment by sobolevn** [comment]
Done! Thanks a lot for your help. 👍

**Pull request merged**